daemon/cmd/daemon_main.go

```diff
@@ -674,6 +674,9 @@ func init() {
 	flags.Int(option.PolicyMapEntriesName, defaults.PolicyMapEntries, "Maximum number of entries in endpoint policy map (per endpoint)")
 	option.BindEnv(option.PolicyMapEntriesName)
 
+	flags.Bool(option.MapEntriesGlobalDynamicSizeName, false, "Enable dynamic sizing of BPF maps based on total system memory")
```

Comment:
The option description (or the doc.) should probably state which maps are affected by this option.

In general, if a user wants to reduce/increase the memory usage after switching dynamic sizing on, do they have a way to know the current sizes?

Reply:
> The option description (or the doc.) should probably state which maps are affected by this option.

Good point, will adjust the usage text.

> In general, if a user wants to reduce/increase the memory usage after switching dynamic sizing on, do they have a way to know the current sizes?

The debug logs will print the calculated map sizes, though there won't be a number in terms of system memory. but users could still use `bpftool` to get the key and value size and estimate the map size using these numbers.